### PR TITLE
Add sysrc resource for managing rc.conf vars on freebsd

### PR DESCRIPTION
**Pull request by kaey**
Also remove special case for sendmail since it might
break somebody's configuration.

### CODE REVIEW

File: resource/sysrc_freebsd.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+// NewSysRC creates a new resource for managing rc.conf variables
+// on a FreeBSD system.
+func NewSysRC(name string) (Resource, error) {
+	s := &SysRC{
+		Base: Base{
+			Name:          name,
+			Type:          "sysrc",
+			State:         "absent",
```
Comment (dnaeon) [inline resource/sysrc_freebsd.go]: Shouldn't this one be `present` by default?

File: resource/sysrc_freebsd.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	}
+
+	if s.Value != v {
+		state.Outdated = true
+	}
+
+	return state, nil
+}
+
+// Create starts the service.
```
Comment (dnaeon) [inline resource/sysrc_freebsd.go]: Please update the doc comments for `Create`, `Delete` and `Update` methods, as the currently they suggest they are for services.

**Comment by dnaeon** [comment]
@kaey thanks for this one!

Please check my comments and if possible fix them, and I'll merge this one!

File: resource/sysrc_freebsd.go (hunk 1 above)
Comment (kaey) [inline resource/sysrc_freebsd.go]: Done.

File: resource/sysrc_freebsd.go (hunk 2 above)
Comment (kaey) [inline resource/sysrc_freebsd.go]: Done.

**Comment by dnaeon** [comment]
Thanks!

**Pull request merged**